File: 프로그래머스0(61~90번)/69.뒤에서 5등 위로.js
// 레벨 0
// https://school.programmers.co.kr/learn/courses/30/lessons/181852

/*
정수로 이루어진 리스트 num_list가 주어집니다.
num_list에서 가장 작은 5개의 수를 제외한 수들을 오름차순으로 담은 리스트를 return하도록 solution 함수를 완성해주세요.
*/

function solution(num_list) {
    return num_list.sort((a,b) => a-b).slice(5)
}

/*
통과못한 코드
function solution(num_list) {
    var answer = [];
    
    let sortedNumbers = num_list.sort((a, b) => a - b); // 오름차순 정렬
    answer = sortedNumbers.slice(5, 10); // 가장 작은 5개 선택

    return answer;
}
*/

File: 프로그래머스 코테(java)/프로그래머스0(61~90번)/69.뒤에서 5등 위로.java
// 레벨 0
// https://school.programmers.co.kr/learn/courses/30/lessons/181852

import java.util.Arrays;

class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = {};
        answer = new int[num_list.length - 5];
        Arrays.sort(num_list);

        for (int i = 0; i < num_list.length; i++) {
            if (i < num_list.length - 5) {
                answer[i] = num_list[i + 5];
            }
        }
        return answer;
    }
}